RiotSharp/Misc/DateTimeConverterFromStringTimestamp.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace RiotSharp
+{
+    class DateTimeConverterFromStringTimestamp : JsonConverter
```

Comment:
Because it's not in an `MM/dd/yyyy HH:mm:ss` format. It's a numeric epoch timestamp encapsulated within a string. Riot is sending it as a string instead of a long. Until they fix that, this is a suitable workaround.